README.md

```diff
@@ -331,6 +331,21 @@ Find full example in [sass-and-ts](examples/sass-and-ts) folder.
 
 </details>
 
+<details id="fast-enough">
+  <summary>Slower than Jest</summary>
+
+When you use `cypress-X-unit-test` for component testing, you might notice the tests are slower than using Jest to test the same components. Yes, that's true. A test runner could be made _extremely_ fast if it did nothing, just check out the [auchenberg/volkswagen](https://github.com/auchenberg/volkswagen) test runner - it is blazing on CI 😉. Of course, Jest does do things, just not inside the real browser environment.
+
+Testing using Jest with its jsdom browser is faster than starting the real browser, loading all libraries, mounting the component and then waiting for the component to actually perform its work in response to the test's actions. But do those tests give you a true confidence that the component is working?
```

Comment:
```suggestion
Testing using Jest with its jsdom browser is faster than starting the real browser, loading all libraries, mounting the component and then waiting for the component to actually perform its work in response to the test's actions. But do those tests give you a true confidence that the component is working?

Try this test 🙈 
> spoiler it fails. [proof](https://codesandbox.io/s/react-testing-library-demo-forked-z7l2o?file=/src/__tests__/components.js)
```js
const mock = jest.fn();
  const { getByRole } = render(
    <button style={{ pointerEvents: "none" }} onClick={mock}>
      text
    </button>
  );

  fireEvent.click(getByRole("button"));
  expect(mock).not.toBeCalled();
 ```
```

https://codesandbox.io/s/react-testing-library-demo-forked-z7l2o?file=/src/__tests__/components.js
```